src/participation.ts

```diff
@@ -87,14 +87,25 @@ export function renderPartipants (dom: HTMLDocument, table: ParticipationTableEl
 export function participationObject (subject: NamedNode, padDoc: NamedNode, me: NamedNode) {
   return new Promise(function (resolve, reject) {
     if (!me) {
-      throw new Error('Not user id')
+      throw new Error('No user id')
     }
 
-    const parps = kb.each(subject, ns.wf('participation')).filter(function (pn) {
-      return kb.holds(pn, ns.wf('participant'), me)
+    const parps = store.each(subject, ns.wf('participation')).filter(function (pn) {
+      return store.holds(pn, ns.wf('participant'), me)
     })
-    if (parps.length > 1) {
-      throw new Error('Multiple records of your participation')
+    if (parps.length > 1) { // This can happen. https://github.com/solid/chat-pane/issues/71
+      const candidates = []
+      for (const participation of parps) {
+        const date = store.anyValue(participation, ns.cal('dtstart'))
+        if (date) {
+          candidates.push([date, participation])
```

Comment:
```suggestion
          candidate = candidate.set(date, participation)
          candidates.push(candidate)
```

Reply:
I used an array so as to be able to sort by date.. surely a set or map doesn't have an order and so wont sort.

Reply:
I was worried about that :(. but I tried to do some other typing but couldn't get it to work.  since it was pushing an array on an array.  I'll do more research.